### DISCUSSION TITLE
Agenda meeting Wed Oct 23 2024

### DISCUSSION THREAD

**Discussion by megmcca** [coreen-mullen/fix_FitSNAP#5] [General]
Agenda:
Create post on Discussion board
- Review command sequence to push changes to git/GitHub
- Go over purpose of each command
- Note: make this so that folks who can’t make it to meetings could reproduce sequence

Create one or more “Issues” to fix
- One real
- One garbage fake made up one
- Note: check status of fix_fitsnap repo (“private” vs “public”)

If time: explain git push/merge conflicts from last meeting
If time: continue Mac vs. PC wars

**Comment by coreen-mullen** [comment]
hi ( professionally )

**Comment by megmcca** [comment]
Ideas for fixes:
- Better logging messages while running the code
- Alyssa mentioned a versioning issue with mpi4py, look into that again
- Hadia has mentioned issues with input file formatting
- Better memory reporting/management since we're now in the era of super huge datasets

**Reply by megmcca** [reply]
see discussion item #6
